client/src/Movies/MovieList.js

```diff
@@ -34,7 +35,7 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
+    <div className="movie-card"> <Link to={`/movies/${movie.id}`} className="movie-card">
```

Comment:
You've got an issue with your rendering that is causing you get to some boxes on your app where it's displaying boxes at the top and bottom of each movie link. See if you can find what's causing it.